src/pybind/mgr/dashboard/controllers/_base_controller.py

```diff
@@ -8,7 +8,7 @@
 import cherrypy
 
 from ..plugins import PLUGIN_MANAGER
-from ..services.auth import AuthManager, JwtManager
+from ..services.auth.auth import AuthManager, JwtManager
```

Comment:
To avoid this ugly redundancy, you usually add the key classes in a directory (subpackage) to the `__all__` variable of the `__init__.py` of that directory. That way you end up importing (which doesn't change the syntax at all):
```suggestion
from ..services.auth import AuthManager, JwtManager
```
This way the new subpackage/module behaves the same as the old one for the rest of the code, but you were able to refactor the into separate files.

Reply:
Right, thanks for the tip. Updated it